README.md

```diff
@@ -45,24 +45,22 @@ You must use the generic model [schema.org](https://schema.org) when you chose t
 We recommend you also serve data in one or more domain models. Domain models make reuse of data much more likely and may contain a wealth of information. Depending on the type of data, you may consider the following examples:
 - Museum collections and catalogues: [CIDOC-CRM](https://cidoc-crm.org) and its derivative [Linked-Art](https://linked.art/model/).
 - Data from archives: [RiC-O](https://www.ica.org/standards/RiC/RiC-O_v0-2.html)
-- Libraries: [BIBFRAME](https://www.loc.gov/bibframe/) or [RDA Elements](https://www.rdaregistry.info/Elements/)
+- Libraries: [RDA Elements](https://www.rdaregistry.info/Elements/) following the recommendations of the [RDA application profile Dutch bibliography (only in Dutch)](https://netwerk-digitaal-erfgoed.github.io/rdanl/). When using other library domain standards like [BIBFRAME](https://www.loc.gov/bibframe/) you should document the implementation choices made and relation to the RDA application profile.
 
-You may also implement any model suggested by [CLARIAH](https://www.clariah.nl) in the [Awesome Ontologies for Digital Humanities](https://github.com/CLARIAH/awesome-humanities-ontologies) repository.
-
-You should implement the [W3C draft for content negotiation by profile](https://www.w3.org/TR/dx-prof-conneg/) when publishing linked data that is available in multiple models.
+When you serve data only in a domain model documentation about the mapping to Schema.org (preferably in machine readable format) should be made available.
 
 #### 2.2 Serving data
 
 The [Implementation Guidelines](https://netwerk-digitaal-erfgoed.github.io/cm-implementation-guidelines/#publishing-collection-information) mentioned earlier consider three levels when [publishing linked data](https://netwerk-digitaal-erfgoed.github.io/cm-implementation-guidelines/#publishing-linked-data):
-1. Basic level: data dumps
-2. Web compliance level: resolvable URIs
+1. Web compliance level: resolvable URIs
```

Comment:
Waarom de volgorde andersom? Is dat niet verwarrend met de tekst zoals beschreven in de Implementation Guidelines? En basic verwacht je toch een beetje als eerste.

Reply:
Ik merk dat er weinig aandacht is voor resolvable URIs terwijl dit de basis is voor linked data. Wat mij betreft is dus compliance met het Web van Data de eerste stap. Het aanbieden van een dump is vooral een convenience ding  voor de hergebruiker, wel belangrijk natuurlijk maar uiteindelijk toch een afgeleide.